.github/workflows/updatecli.yaml

```diff
@@ -38,6 +38,55 @@ jobs:
         with:
           commit_message: |
             🛠 Updatecli pipeline bump
-          branch: ${{ github.event_name == 'workflow_dispatch' && 'updatecli-bump' || '' }}
+          branch: ${{ github.event_name == 'workflow_dispatch' && 'updatecli-bump-helm' || '' }}
+          create_branch: ${{ github.event_name == 'workflow_dispatch' }}
+          push_options: ${{ github.event_name == 'workflow_dispatch' && '--force' || '' }}
+
+  bump-acs-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+
+      - uses: >-
+          Alfresco/alfresco-build-tools/.github/actions/get-branch-name@v1.41.0
+
+      - name: Login to quay.io
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+
+      - name: Install Updatecli
+        uses: updatecli/updatecli-action@v2
+        with:
+          version: v0.54.0
+
+      - name: Checkout updatecli configs
+        uses: actions/checkout@v3
+        with:
+          repository: alfresco/alfresco-updatecli
+          ref: OPSEXP-2173-andromeda
```

Comment:
revert to master after https://github.com/Alfresco/alfresco-updatecli/pull/1

Reply:
Actually I like th idea to have "sprints" tags in this repo.

Reply:
we can make a tag on alfresco-updatecli once everything is merged and seems working